odl/trafos/wavelet.py

```diff
@@ -286,6 +286,28 @@ def _call(self, x):
         else:
             raise RuntimeError("bad `impl` '{}'".format(self.impl))
 
+    def scales(self):
```

Comment:
The design here could probably be improved. Perhaps this can somehow be moved to the Base class. In addition to that, this should have a doctest to show more clearly what the scales actually are.